WordPress/src/main/java/org/wordpress/android/ui/posts/navigation/EditPostDestination.kt

```diff
@@ -0,0 +1,35 @@
+package org.wordpress.android.ui.posts.navigation
+
+/**
+ * Represents the available destinations in the Edit Post flow.
+ * Provides type-safe navigation between different screens in the post editing interface.
+ */
+sealed class EditPostDestination {
+    /**
+     * The main editor screen where users write and edit post content.
+     */
+    object Editor : EditPostDestination()
```

Comment:
NP: AS recommends converting these `objects` into `data objects`.

<img width="545" height="153" alt="data" src="https://github.com/user-attachments/assets/1d0f06bf-adb8-48ab-bcd7-5c2036542ff7" />

Reply:
Interesting that my AS doesn't 😕 

Addressed in 662b71a87eb55dea66bed19fd231c367ab675acf in #22068.